### DISCUSSION TITLE
Dexcom G6 „Firmware Version: 2.27.2.105 unknown“

### DISCUSSION THREAD

**Discussion by Brombrax** [NightscoutFoundation/xDrip#3785] [Q&A] [answered]
Looks like the firmware of my new dexcom g6 transmitter is unknown. Xdrip reacts unusual when calibrating, it requests two values for calibration. Worked well before I changed the transmitter yesterday. Is it save to use nightly version and will this fix the problem?
Thx, Brom
![IMG_1038](https://github.com/user-attachments/assets/dcd7d5a6-348d-4611-83ee-deb7c54ff512)

**Comment by Navid200** [comment, marked as answer]
Yes, you need to update to the Nightly 2024.10.29 or any one after that.  
https://navid200.github.io/xDrip/docs/ReleaseNotes.html
